recipes-bsp/opensbi/opensbi_%.bbappend

```diff
@@ -0,0 +1,16 @@
+# Mainline OpenSBI supports the C906 out of the box, but it needs a few tweaks
+# and a new reset driver for the sunxi watchdog.
+FILESEXTRAPATHS:prepend:nezha := "${THISDIR}/files:"
+
+SRC_URI:append:nezha = " \
+    file://0001-lib-utils-fdt-Require-match-data-to-be-const.patch \
+    file://0002-lib-utils-timer-Add-a-separate-compatible-for-the-D1.patch \
+"
+
+EXTRA_OEMAKE:nezha = "PLATFORM=${RISCV_SBI_PLAT} I=${D} FW_PIC=y CLANG_TARGET= "
+
+do_install:prepend:nezha () {
+    autotools_do_install
```

Comment:
why is this needed ?

Reply:
I checked and it is not used so I removed it.